api_app/connectors_manager/helpers.py

```diff
@@ -2,15 +2,45 @@
 # See the file 'LICENSE' for copying permission.
 
 from django.conf import settings
+from django.core.cache import cache
 
 import os
 import json
+import logging
 
+from api_app.connectors_manager.serializers import ConnectorConfigSerializer
 
-def get_connector_config():
+
+logger = logging.getLogger(__name__)
+
+
+def get_verified_connector_config():
+    # get cached json else cache and return
+    config = cache.get("verified_connectors_config", None)
```

Comment:
instead of `cache.get` or `cache.set` use the `@cache_memoize` decorator from https://github.com/peterbe/django-cache-memoize. The package is already included in IntelOwl.